cairo/tests/ethereum/cancun/vm/test_interpreter.py

```diff
@@ -0,0 +1,25 @@
+from hypothesis import given
+
+from ethereum.cancun.vm import Environment, Message
+from ethereum.cancun.vm.interpreter import execute_code
+from ethereum.exceptions import EthereumException
+from tests.utils.errors import strict_raises
+from tests.utils.message_builder import MessageBuilder
+from tests.utils.strategies import environment_lite
+
+
+class TestInterpreter:
+    @given(message=MessageBuilder().build_full_message(), env=environment_lite)
```

Comment:
you need to (temporarily) modify your strategy so that it doesn't addresses of precompiles; otherwise the output will be different